src/main/app/claims/ccjClient.ts

```diff
@@ -1,25 +1,18 @@
 import { User } from 'idam/user'
 import { request } from 'client/request'
-import { CCJModelConverter } from 'claims/ccjModelConverter'
 import { claimStoreApiUrl } from 'claims/claimStoreClient'
 import { CountyCourtJudgment } from 'claims/models/countyCourtJudgment'
 import { Claim } from 'claims/models/claim'
-import { DraftCCJ } from 'ccj/draft/draftCCJ'
-import { Draft } from '@hmcts/draft-store-client'
 
 export class CCJClient {
 
-  static async request (externalId: string, draft: Draft<DraftCCJ>, user: User): Promise<Claim> {
-    const countyCourtJudgment: CountyCourtJudgment = CCJModelConverter.convertForRequest(draft.document)
-    return CCJClient.save(externalId, countyCourtJudgment, user, false)
-  }
-
-  private static async save (externalId: string, countyCourtJudgment: CountyCourtJudgment, user: User, issue: boolean = false): Promise<Claim> {
+  static async request (externalId: string, countyCourtJudgment: CountyCourtJudgment, user: User, issue: boolean): Promise<Claim> {
```

Comment:
`issue: boolean` last parameter is not needed anymore.

Reply:
Also make sure `ccjType` is set to `ADMISSIONS' for admissions journey and not hardcoded as `DEFAULT`